packages/parity-electron/README.md

```diff
@@ -56,14 +54,16 @@ Returns the path to the Parity path inside Electron's `userData` folder, even if
 
 Resolves to `true` if Parity is currently running, or to `false` if not.
 
-#### `runParity(additionalFlags: Array<String>, onParityError: Function): Promise<Null>`
+#### `runParity(options: Object): Promise<Null>`
 
 Spawns a child process to run Parity. If some `cli` flags are passed into the options in `parityElectron`, then those flags will be passed down to Parity itself.
 
-| Option            | Type            | Description                                                                                   |
-| ----------------- | --------------- | --------------------------------------------------------------------------------------------- |
-| `additionalFlasg` | `Array<String>` | Addtional flags to pass to Parity, listed as an array, to be passed to `child_process.spawn`. |
-| `onParityError`   | `Function`      | Callback with `error` as argument when Parity encounters an error.                            |
+| Option                    | Type            | Description                                                                                                                    |
+| ------------------------- | --------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| `options.flags`           | `Array<String>` | Additional flags to pass to Parity, listed as an array, to be passed to `child_process.spawn`.                                 |
+| `options.wsInterface`     | `String`        | Hostname portion of the WebSockets server Fether will connect to. It should be an interface's IP address. (default: 127.0.0.1) |
+| `options.wsPort`          | `String`        | Port portion of the WebSockets server Fether will connect to. (default: 8546)                                                  |
```

Comment:
What if I pass different `option.wsInterface` and `--ws-interface` in flags?

I think having wsInterface and wsPort as options is a bit too confusing here. I would prefer having the code look for `--ws-interface` in the flags, and taking the next element in the array, and then passing it down to `isParityRunning`.